core/src/test/scala/unit/kafka/message/MessageTest.scala

```diff
@@ -66,26 +65,31 @@ class MessageTest extends JUnitSuite {
       } else {
         TestUtils.checkEquals(ByteBuffer.wrap(v.payload), v.message.payload)
       }
+
       // check timestamp
-      if (v.magicValue > Message.MagicValue_V0)
+      if (v.magicValue > Message.MagicValue_V0) {
         assertEquals("Timestamp should be the same", v.timestamp, v.message.timestamp)
-       else
+      } else {
         assertEquals("Timestamp should be the NoTimestamp", Message.NoTimestamp, v.message.timestamp)
+      }
```

Comment:
I'm aware of all the pros/cons. Opinions on this vary depending on the language. In Scala, it's very common for if/else to be written without braces and this is Scala code. Some languages rely purely on significant whitespace and braces are not even an option.

Code is read much more often than it's changed, so we generally optimise for that. If people need to add a brace for debugging, they can do that.

Finally, for subjective things like this, it's good etiquette to follow the project guidelines. There is no right or wrong.